cards/packs.go

```diff
@@ -0,0 +1,12 @@
+package cards
+
+type Pack struct {
+	Rare Card
+	Uncommons [3]Card
+	Commons [10]Card
+	Foil Card
+}
+
+type PackGenerator interface {
+	GetPack() Pack
```

Comment:
Kinda nit-picky, but maybe rename this to `Generate` or `GeneratePack`, as it's a bit more descriptive about what it's supposed to do.